### ISSUE TITLE
Allow saving whiteboard to a new note

### ISSUE THREAD

**Issue by xmha97** [adil192/saber#344]
How can I save whiteboard tab to browse tab?
![Screenshot_2023-02-13-17-28-06-699_com.adilhanney.saber.jpg](https://user-images.githubusercontent.com/23485114/218479298-2ed86479-7875-495c-a173-712e20c1c120.jpg)

**Comment by memartinb** [comment]
Came across that missing, too, so definitely +1 !

Cheers!

**Comment by adil192** [comment]
The whiteboard is intended to be a place for temporary quick scribbles (such as doing some practice questions that you don't want to keep) so it doesn't make sense for it to go with your actual (long-term) notes. This is in the same sense that you wouldn't keep a physical whiteboard alongside your paper notes.

**Comment by memartinb** [comment]
I'd like to object that more often than not, even physical whiteboards are photographed (saved!) before wiping; also on digital whiteboards there are widely used funktions to save and/or share the work after a meeting, class, etc.

I understand that the whiteboard function in saber is intended to be only temporary, yet I suppose there is or will be a hardly deniable demand in users to "change their mind" after a quick scribble to keep the work, which is not possible right now (neither select and copy to note).

So I am sure a small "Save as new note" Button wouldn't hurt, and might be a function / use case that many users will be looking for in Saber.

**Comment by smellymoo** [comment]
+1

It doesn't make conceptial sense, but it does make practical sense...

Say you draw something to test an idea, then you like it and want to save it. Why not? It's cheap to add, no downside and would improve the UX for some people.

**Comment by xmha97** [comment]
> +1
> 
> It doesn't make conceptial sense, but it does make practical sense...
> 
> Say you draw something to test an idea, then you like it and want to save it. Why not? It's cheap to add, no downside and would improve the UX for some people.

Exactly 👌

**Comment by ceskyDJ** [comment]
I agree with older comments. This is really a good idea. Sometimes you just want to draw details of some idea and thinking it is just one of non-working concepts BUT when you finish your mind map / prototype / ..., you realize it is really good and you want to at least save it for future work. Similar situation happens when you are planning something with someone and at the end, both of you want to save ideas presented on the whiteboard (e.g. from meeting for some project) just for sure you won't forget them.

**Comment by xmha97** [comment]
> I agree with older comments. This is really a good idea. Sometimes you just want to draw details of some idea and thinking it is just one of non-working concepts BUT when you finish your mind map / prototype / ..., you realize it is really good and you want to at least save it for future work. Similar situation happens when you are planning something with someone and at the end, both of you want to save ideas presented on the whiteboard (e.g. from meeting for some project) just for sure you won't forget them.

Thanks for your reply 🙏

**Comment by tasmo** [comment]
I am honestly curious why in such a thoughtful app this option is missing yet.

**Comment by ceskyDJ** [comment]
I think @adil192 started to implement new features regarding to votes (number on thumbs up reactions), so maybe voting could help a bit 😅.

**Comment by gfkaoshdnal** [comment]
Logged in just to upvote this lol
I'm on Android and just installed the app, there is a Save button in Whiteboard but where does it go when you save? I'm fearful of toggling the Auto-clear Whiteboard option...
I use Whiteboard because it's a 1 tap option. A new note requires 2 (+ then New note).
